policies/bundles/L3FirewallEnabled/metric.rego

```diff
@@ -13,5 +13,6 @@ applicable {
 }
 
 compliant {
+	compare("isIn",  "NetworkInterface", input.type)
```

Comment:
When I think about it, I'm for putting the `input.type` check to the applicable rule. Otherwise, the metric is applied to L3WebFirewall as well what we want to avoid here in the first place.

Reply:
Done